### DISCUSSION TITLE
Missing the copy raw botton

### DISCUSSION THREAD

**Discussion by wep13** [community/community#138938] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I don't understand why in this file i am missing the copy raw botton
![image](https://github.com/user-attachments/assets/f73d2a1a-f461-493a-900f-2eaf550b23a7)

**Comment by jovanovic-djo** [comment]
At the first moment, i thought that .ipynb files cannot be copied raw, but here is my example.
![image](https://github.com/user-attachments/assets/ed311d11-b719-4cee-a539-78ed7c504bab)
As you can see, I can copy it.

Here is explanation for your situation:
https://stackoverflow.com/questions/52380244/github-pencil-edit-button-disappears

TLDR - Its due to size of your file.
